models/generated/workloads/vm.go

```diff
@@ -0,0 +1,103 @@
+package workloads
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"math"
+	"net"
+
+	"github.com/rs/zerolog/log"
+	schema "github.com/threefoldtech/tfexplorer/schema"
+)
+
+var _ Workloader = (*VirtualMachine)(nil)
+var _ Capaciter = (*VirtualMachine)(nil)
+
+type VirtualMachine struct {
+	ReservationInfo `bson:",inline"`
+
+	Flist     string   `bson:"flist" json:"flist"`
+	NetworkId string   `bson:"network_id" json:"network_id"`
+	Ipaddress net.IP   `bson:"ipaddress" json:"ipaddress"`
+	SshKeys   []string `bson:"ssh_keys" json:"ssh_keys"`
+	// why isn't this a net.IP? because it's a wid
+	PublicIP schema.ID  `bson:"public_ip" json:"public_ip"`
+	Capacity VMCapacity `bson:"capcity" json:"capacity"`
```

Comment:
I am vague about the requirements of this but may be we should stick to k8s sizes. IMHO a k8s machine should be a "special" VirtualMachine, since they have so much in common a k8s struct can inherit from a VirtualMachine. This will make it much easier to reuse current code in ZOS.